tests/Php72/Php72Test.php

```diff
@@ -64,4 +64,26 @@ public function testSplObjectId()
 
         $this->assertNull(@spl_object_id(123));
     }
+
+    /**
+     * @covers Symfony\Polyfill\Php72\Php72::sapi_windows_vt100_support
+     */
+    public function testSapiWindowsVt100Support()
+    {
+        if ('\\' !== DIRECTORY_SEPARATOR) {
+            $this->markTestSkipped('Windows only test');
+        }
+
+        $this->assertFalse(p::sapi_windows_vt100_support(STDOUT, false));
```

Comment:
we should make the test pass without the `p::` prefix: the testing framework will ensure the test is run twice: with and without the polyfill when possible, so that behavior parity is "proven" on 7.2

Reply:
we do know it, but as you see it's not obvious, maybe it worth to put that knowledge somewhere? (contributing file perhaps?)

Reply:
@keradus Why not :) The readme has this sentence on the topic:

> Polyfills are unit-tested alongside their native implementation so that feature and behavior parity can be proven and enforced in the long run.

Feel free to submit a PR to amend the README.md, or another file that could be a better fit to you.